settings.gradle

```diff
@@ -0,0 +1,13 @@
+rootProject.name = 'protoactor-kotlin'
+
+include 'proto.mailbox'
+include 'proto.actor'
+include 'proto.router'
+include 'proto.remote'
+include 'examples'
+
+project(':proto.mailbox').name = 'proto-mailbox'
```

Comment:
This is needed since the publish plugin uses the project name as the `artifact-id` by convention.
Changing the folder names for the sub-projects works but the change is too big at this point.
Moving the `artifact-id` to `ArtifactExtension` is another option.